www/assets/js/lib/search.ts

```diff
@@ -103,33 +108,60 @@ export const resourceJSONToLearningResource = (
   }
 }
 
-export const searchResultToLearningResource = (
-  result: LearningResourceResult
+export const courseSearchResultToLearningResource = (
```

Comment:
We should have a follow up pr to get rid of the LearningResource type defined in ocw-hugo-themes update the cards in `SearchResult` to expect the objects returned from the search api directly and update courseJSONToLearningResource and resourceJSONToLearningResource to convert ocw json to CourseResource and ContentFile from the open api.